pkg/apis/navigator/validation/elasticsearch_test.go

```diff
@@ -360,37 +378,59 @@ func TestValidateElasticsearchClusterSpec(t *testing.T) {
 			Plugins:        validSpecPluginsList,
 			NodePools:      []navigator.ElasticsearchClusterNodePool{newValidNodePool("test", 3, navigator.ElasticsearchRoleMaster)},
 			Pilot:          validSpecPilotImage,
-			Image:          validSpecESImage,
+			Image:          &validSpecESImage,
 			MinimumMasters: 1,
 		},
 		"minimum masters greater than total number of masters": {
 			Plugins:        validSpecPluginsList,
 			NodePools:      []navigator.ElasticsearchClusterNodePool{newValidNodePool("test", 3, navigator.ElasticsearchRoleMaster)},
 			Pilot:          validSpecPilotImage,
-			Image:          validSpecESImage,
+			Image:          &validSpecESImage,
 			MinimumMasters: 5,
 		},
+		"missing elasticsearch version": {
+			Plugins:   validPluginsList,
+			NodePools: []navigator.ElasticsearchClusterNodePool{validNodePool, validNodePool},
+			Pilot:     validPilotImage,
+			Image:     validESImage,
+		},
```

Comment:
Version is missing from all the other error cases too. 
So it's not clear whether they're failing because of version or minimummaster